lib/zendesk_api/collection.rb

```diff
@@ -410,11 +435,13 @@ def get_response(path)
       end
     end
 
-    def handle_response(response_body)
+    def handle_response(response_body, path = nil)
```

Comment:
Lets use  keyword arguments instead of positional arguments.

```
def foo(bar: "default")
  puts bar
end

foo # prints "default"
foo(bar: "baz") # prints "baz"
```

The reason of this is because using keyword arguments increases flexibility as we can change the order for example vs traditional params with a default value.

Reply:
Dismiss this based on my previous comment where we should move this away.